testsuite/unittests/test_aeif_cond_beta_multisynapse.sli

```diff
@@ -0,0 +1,162 @@
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+ 
+ /* BeginDocumentation
+   Name: testsuite::test_aeif_cond_beta_multisynapse - sli script for the aeif multisynapse model with synaptic conductance modeled by a double exponential function
+
+   Synopsis: (test_aeif_cond_beta_multisynapse) run
+
+   Description:
+   This test creates a multisynapse neuron with three receptor ports with
+   different synaptic rise times and decay times, and connect it to
+   two excitatory and one inhibitory signals. If the rise times and decay
+   times are much greater than the membrane time constant, the difference
+   between the membrane potential and the rest potential V(t) will be
+   approximately proportional to the synaptic conductance g(t), which can
+   be computed analitically. At the end, the script compares the simulated
+   values of V(t) with the approximate theoretical prediction.
+   
+   SeeAlso:
+
+   FirstVersion: August 2016
+   Author: Bruno Golosio
+ */
+ 
+(unittest) run
+/unittest using
+
+0
+<< 
+    /overwrite_files true
+>> SetStatus
+
+% This test should only run if we have GSL
+statusdict/have_gsl :: not
+{
+  exit_test_gracefully
+}
+if
+
+/tolerance   	5e-2 def %mV
+/simulation_t	2500 def %ms
+/V_peak			0. def
+/a 				4. def
+/b 				80.5 def
+/taus_decay		[400.0 200.0 200.0] def % synaptic decay times  
+/taus_rise		[200.0 100.0 100.0] def % synaptic rise times  
+/weights		[0.1 0.1 -0.1] def % synaptic weights
+/delays     	[1. 1000. 1500.] def % ms - synaptic delays
+/spike_time 	10. def
+/dt         	0.1 def % time step
+/Vrest -70.6 def % resting potential
+/g_L 30.0 def
+/E_in -85.0 def
+
+/spike_generator Create /sg Set
+sg << /spike_times [spike_time] >> SetStatus % generates a single peak
+
+
+% Creates a voltmeter
+/voltmeter Create /vm Set
+vm << /withtime true /time_in_steps true /interval dt /to_file true /precision 5 >> SetStatus
+
+% Creates one aeif_conf_beta_multisynapse neuron
+/aeif_cond_beta_multisynapse Create /multisynapse_neuron Set
+multisynapse_neuron << /taus_decay taus_decay >> SetStatus
+multisynapse_neuron << /taus_rise taus_rise >> SetStatus
+multisynapse_neuron << /V_peak V_peak >> SetStatus
+multisynapse_neuron << /a a >> SetStatus
+multisynapse_neuron << /b b >> SetStatus
+multisynapse_neuron << /E_L Vrest >> SetStatus
+multisynapse_neuron << /g_L g_L >> SetStatus
+multisynapse_neuron << /E_in E_in >> SetStatus
+
+multisynapse_neuron ShowStatus
```

Comment:
Useful for development, but I suggest to remove it since it will not be of use in the actual test.